prototype/mobile_endpoint/backends/couch/models.py

```diff
@@ -0,0 +1,38 @@
+from couchdbkit import Document
+from mobile_endpoint.form.models import XFormInstance
+from mobile_endpoint.models import ToFromGeneric
+
+
+class CouchForm(Document, ToFromGeneric):
```

Comment:
Is this necessary? Could you not just make the XFormInstance doc inherit from Document?